src/null-safety/index.md

```diff
@@ -115,6 +115,24 @@ To resolve these issues, check for
 of any packages you installed from pub.dev, and [migrate](#migrate) all
 of your own source code to use sound null safety.
 
+As of late January 2023, **Dart 3 alpha** is available from the Dart dev channel
```

Comment:
This section got pretty large. Consider putting it in a subheader.

```suggestion
### Dart 3 alpha

As of late January 2023, **Dart 3 alpha** is available from the Dart dev channel
```

Reply:
I want this to be prominent, so keeping. It's only for some months anyway